### DISCUSSION TITLE
Query Log > Unblock:

### DISCUSSION THREAD

**Discussion by yanniedog** [AdguardTeam/AdGuardHome#2733] [Q&A]
Nothing unblocks on the first try. Even after selecting "unblock", it still blocks the same item a second time.

When I unblock the same item after the second block, it works correctly.

Any reason for this?

**Comment by ameshkov** [comment]
> Nothing unblocks on the first try. Even after selecting "unblock", it still blocks the same item a second time.

Why do you think it still blocks it? It could be that the client uses the cached blocked response (it's cached for 30 seconds)
